src/contexts/apiV2sendSearchRequest.ts

```diff
@@ -64,34 +66,50 @@ type Config = {
   };
 };
 
-const convertReranker = (reranker?: Config["search"]["reranker"]) => {
-  if (!reranker) return;
-
-  if (reranker.type === "none") {
+const convertReranker = (reranker?: Config["search"]["reranker"]): NoneReranker | ChainReranker => {
+  if (reranker?.isEnabled) {
     return {
-      type: reranker.type
-    };
-  }
+      type: "chain",
+      rerankers: reranker?.names?.split(",").map((name: string) => {
+
+        switch (name) {
```

Comment:
So is reranker always created as a type "chain" in our API, even if it's just a single reranker?